### PR TITLE
Back-port #45667 to 2017.7.3

### PR DESCRIPTION
**Pull request by rallytime**
Back-port #45667 to 2017.7.3

### CODE REVIEW

File: salt/modules/pacman.py [hunk 1]
```diff
@@ -686,7 +684,7 @@ def upgrade(refresh=False, root=None, **kwargs):
             and __salt__['config.get']('systemd.scope', True):
         cmd.extend(['systemd-run', '--scope'])
     cmd.extend(['pacman', '-Su', '--noprogressbar', '--noconfirm'])
-    if salt.utils.is_true(refresh):
+    if refresh:
```
Comment (terminalmage) [inline salt/modules/pacman.py]: If we're not using `salt.utils.is_true()` any more, then we should be using `if refresh is True` here.

File: salt/modules/pacman.py [hunk 2]
```diff
@@ -478,7 +478,8 @@ def install(name=None,
 
     sysupgrade
         Whether or not to upgrade the system packages before installing.
-
+        If refresh is set to ``True`` but sysupgrade is not set, ``-u`` will be
```
Comment (terminalmage) [inline salt/modules/pacman.py]: I would prefer to use `specified` instead of `set` here.

**Review by terminalmage — CHANGES_REQUESTED** [review]

**Comment by terminalmage** [comment]
I didn't realize that this was a backport PR for https://github.com/saltstack/salt/pull/45667, I reviewed it first. We should probably have @gtmanfred make the changes in https://github.com/saltstack/salt/pull/45667 and then cherrypick them onto this PR once he's done that.

**Comment by rallytime** [comment]
@terminalmage Yeah, sounds good. @gtmanfred LMK when that's updated on your end and I'll pull in your commit.

**Review by terminalmage — APPROVED** [review]

**Pull request merged**